README.md

```diff
@@ -275,6 +280,176 @@ Reads if device geolocation is activated or not (read-only).
 console.log(pk.hasGeolocation); // true or false
 ```
 
+### `pk.patch.list()`
+
+List, filter and paginate patches.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// get all patches, paginated
+pk.patch.list().then((res) => {
+  console.log(res.results);
+});
+
+// filter and paginate patches
+pk.patch.list({
+  query: 'angeles',
+  countries: ['us'],
+  types: ['street'],
+  status: 'approved',
+  maxResults: 10,
```

Comment:
Missing `language`

Reply:
Actually, removed some attributes, the point is just to show an example.